apps/ai_reports/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from adhocracy4.comments.models import Comment
+from adhocracy4.models.base import TimeStampedModel
+
+
+class AiReport(TimeStampedModel):
+    is_pending = models.BooleanField(default=True)
+    category = models.CharField(max_length=50)
+    explanation = models.TextField()
+    confidence = models.IntegerField(default=0)
```

Comment:
My reflex was to use an int as in Euro cents or percentage, to avoid strange rounding / comparison behaviors. But yeah, let's go for float. I'll change it to `models.FloatField(default=0)`.

Reply:
I guess it depends on what we will actually receive from the AI. The designs suggest it's an int but that might have been a guess as well. Maybe we ask the pms if they know it

Reply:
ok, I changed it to `FloatField` in the latest version. For now this is only a dummy model, let's migrate it once we know what we actually receive / want to store.